packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css

```diff
@@ -15,6 +15,24 @@
   --width-left-navigation-when-expanded: 256px;
 }
 
+@keyframes visible {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes hidden {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+}
```

Comment:
I don't think it's used anywhere

Reply:
Thank you,
All cleaned up here: https://github.com/commercetools/merchant-center-application-kit/pull/3237/commits/8dc2680475702e90085f24a037efa476469e5290